import_streets/src/extract.rs

```diff
@@ -119,15 +119,14 @@ impl OsmExtract {
             return false;
         }
 
-        // If we're only handling sidewalks tagged on roads, skip a bunch of ways
+        // If we're only handling sidewalks tagged on roads, skip crossings and separate sidewalks
+        // Note we have to do this here -- get_lane_specs_ltr doesn't support decisions like
+        // "actually, let's pretend this road doesn't exist at all"
         if opts.map_config.inferred_sidewalks {
```

Comment:
So this experiment now just means "bring in `highway=footway, footway={sidewalk,crossing}` ways". All of the new stuff in this PR works both with and without this experiment